packages/blocks/src/__internal__/utils/selection.ts

```diff
@@ -68,22 +68,22 @@ function fixCurrentRangeToText(
   return range;
 }
 
-export function focusRichText(
+async function awaitUntilNextTick() {
```

Comment:
I think this is not "next tick" but rather a so called "macro task". Why not reuse the `sleep` util here? https://github.com/toeverything/blocksuite/blob/master/packages/blocks/src/__internal__/utils/std.ts#L51

Reply:
Also, just saying that we don't have to reply on each review comment, some comments that be easily resolved can be directly marked.